src/main/java/gregtech/api/capability/impl/MultiblockFuelRecipeLogic.java

```diff
@@ -19,29 +16,34 @@ public MultiblockFuelRecipeLogic(RecipeMapMultiblockController tileEntity) {
         super(tileEntity);
     }
 
+    @Nonnull
     @Override
     protected int[] runOverclockingLogic(@Nonnull IRecipePropertyStorage propertyStorage, int recipeEUt, long maxVoltage, int recipeDuration, int amountOC) {
+        // no overclocking happens other than parallelization,
+        // so return the recipe's values, with EUt made positive for it to be made negative later
+        return new int[]{-recipeEUt, recipeDuration};
+    }
+
+    @Override
+    protected void modifyOverclockPre(@Nonnull int[] values, @Nonnull IRecipePropertyStorage storage) {
```

Comment:
Should this call super like the other calls?

Reply:
This should not. Fuel logic needs the duration to act opposite to how regular recipes operate.

A regular recipe having a duration decrease makes it faster, but with a generator would produce less energy, so instead the generator uses division to make it longer, therefore producing more power instead.

The inverse is also true, a longer recipe multiplier from the hatch should make generators have a shorter recipe (less power generated) in exchange for the bonus.